compiler/luci/service/src/CircleShapeInferenceRule.cpp

```diff
@@ -1478,6 +1478,16 @@ class ShapeInferenceAlgorithm final : public luci::CircleNodeVisitor<loco::NodeS
     return loco::NodeShape{input_shape};
   }
 
+  loco::NodeShape visit(const luci::CircleReverseV2 *node) final
+  {
+    auto input_shape = loco::shape_get(node->tensor()).as<loco::TensorShape>();
+    auto axis_shape = loco::shape_get(node->axis()).as<loco::TensorShape>();
```

Comment:
`axis_shape `  is only used for `assert`, which will fail in release build.

Can you check this is working?
```cxx
LUCI_ASSERT(loco::shape_get(node->axis()).as<loco::TensorShape>().rank() == 1, "Tensor must be 1-D");
```